templates/base/Dockerfile

```diff
@@ -3,10 +3,33 @@ FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes build-essential ca-certificates jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip wget && \
```

Comment:
@Wykenz, please make sure that the new packages added here don't have vulnerabilities on the installed versions